docs/source/rectify.md

```diff
@@ -0,0 +1,145 @@
+# Spatial Rectification Algorithm
+
+This chapter describes the algorithm used in the function [`rectify_dataset()`](api.html#xcube.core.resampling.rectify_dataset) 
+of module `xcube.core.resampling`. The function geometrically transforms 
+spatial [data variables](https://docs.xarray.dev/en/stable/user-guide/terminology.html#term-Variable) 
+of a given [dataset](https://docs.xarray.dev/en/stable/user-guide/terminology.html#term-Dataset) 
+from an irregular source 
+[grid mapping](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.11/cf-conventions.html#grid-mappings-and-projections) 
+into new data variables for a given regular target grid mapping and returns 
+them as a new dataset. 
+
+## Problem Description
+
+The following figure shows a Sentinel-3 OLCI Level-1b scene in its original 
+satellite perspective. In addition to the measured reflectances, the data 
+product also provides the latitude and longitude for each pixel as two 
+individual coordinate images. To the right, the absolute value of the 
+gradient vectors  of the latitude and longitude  *|(∇ lon)² + (∇ lat)²|* 
+are shown:  
+
+<!-- <img src="rectify/olci-input.png" alt="OLCI L1B Input" style="width:60em;display:block;margin-left:auto;margin-right:auto"/> -->
+![OLCI L1B Input](rectify/olci-input.png)
+
+The given coordinates, latitude and longitude, are _terrain corrected_ with 
+respect to a digital elevation model (DEM) that represents the Earth's geoid.
```

Comment:
I'm confused about the use of "geoid" here and in the rest of the text and diagram. I only know "geoid" as an equipotential gravitational surface which doesn't correspond completely to any set of physical features (as illustrated e.g. [here](https://www.britannica.com/science/geoid) ). But in this text "geoid" seems to be synonymous with "elevation".

Reply:
Ok, convinced, I will use just DEM.